homeassistant/components/nextdns/strings.json

```diff
@@ -25,5 +25,294 @@
     "info": {
       "can_reach_server": "Reach server"
     }
+  },
+  "entity": {
+    "binary_sensor": {
+      "this_device_nextdns_connection_status": {
+        "name": "This device NextDNS connection status"
```

Comment:
Why not make this "Device connection status"?

Reply:
Good idea. I'll open a PR with those two changes.